src/App.jsx

```diff
@@ -1,3 +1,4 @@
+import { TwitterTweetEmbed, TwitterTimelineEmbed } from 'react-twitter-embed';
 import { useEffect, useState } from 'react';
 import axios from 'axios';
 import dotenv from 'dotenv';
```

Comment:
Useless imports

Reply:
And that makes me think: How did Prettier/ESlint let it go ?

Reply:
We need it for the management of the order of the sections, seen in a previous PR